jest/jest.setup-dom.js

```
@@ -55,3 +55,16 @@ Object.defineProperty(
+
+class MockBroadcastChannel {
+  constructor() {
+    this.onmessage = null;
+  }
+  postMessage() {}
+  close() {}
+}
+global.BroadcastChannel = MockBroadcastChannel;
+
+global.TransformStream = class {
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)